app/Http/Controllers/TableController.php

```diff
@@ -60,6 +61,34 @@ public function edit(Request $request)
         return view('table.edit', compact('table'));
     }
 
+    public function getTable(Request $request)
+    {
+        $table = $this->tableRepository->get($request->table_id);
+        $table->cost = number_format($table->cost, 2);
+        return json_encode($table);
+    }
+
+    public function getTables(Request $request)
+    {
+        $user = Auth::user()->id . Auth::user()->username;
+        $tables = $request->session()->get($user, function () {
+            return array();
+        });
+        $ids = array_values($tables);
+        $bookedTables = array_map(function ($id) { return $this->tableRepository->get($id);}, $ids);
+        array_map(function ($bookedTable) {$bookedTable->cost = number_format($bookedTable->cost, 2);}, $bookedTables);
```

Comment:
Do the same, as recommended previously. Always avoid stacking things on one line even, esp. with functions that all call-backs.